Add .Net 6 to TargetFrameworks

jeremiedcyr opened the pull request:
.Net 5 is out of support so I added .Net 6 which is in Long time support.

We want to use this library but the fact that it is targeting .Net 5 could become a liability as we are targeting .Net 6 in our code base.

Commit:
Add net6.0 in TargetFrameworks

Commit:
Add net6.0 in build and publish workflows

guyglies100 commented on Mongo.Migration/Mongo.Migration.csproj:
```diff
@@ -4,7 +4,7 @@
     <RootNamespace>Mongo.Migration</RootNamespace>
     <Version>3.1.4</Version>
     <PackageVersion>3.1.4</PackageVersion>
```
💡 On devrait bumper la version.

Commit:
build: updated deps

Commit:
feat: adde azdo PR pipeline

Commit:
fix: added OP process

Commit:
build: added ci-cd pipeline defintion

Commit:
build: added nuget config

Commit:
build: updated package generation

Commit:
fix: code cleanup

Commit:
build: added minver

Commit:
fix: updated description

Commit:
build(deps): updated ms packages

Commit:
fix: removed regions

delepster committed:
feat: Office Protect process (#1)

Commit:
fix: update serializer registration

delepster committed:
fix: update serializer registration

delepster committed:
fix: lower trace level for no migrations found (#3)

Commit:
build: updated deps

filipe-pinto-deltatre commented:
It would be great if this is merged... would solve a lot of issues for me...

delepster committed:
fix: register startup migration explicitly (#4)

pleveille333 committed:
Add MongoMigrationHostedService

pleveille333 committed:
Merge branch 'master' into feat/AddMongoMigrationHostedService

pleveille333 committed:
Use StopApplication when migration.Run fail

pleveille333 committed:
Downgrade Microsoft.Extensions.Hosting to 6.0.1

pleveille333 committed:
feat: Merge pull request #5 from sherweb/feat/AddMongoMigrationHostedService

feat: Add MongoMigrationHostedService

Commit:
build: updated deps

Commit:
build: updated deps

dependabot[bot] committed:
build: Bump MongoDB.Driver from 2.8.0 to 2.19.0 in /Mongo.Migration.Test (#6)

Bumps [MongoDB.Driver](https://github.com/mongodb/mongo-csharp-driver) from 2.8.0 to 2.19.0.
- [Release notes](https://github.com/mongodb/mongo-csharp-driver/releases)
- [Commits](https://github.com/mongodb/mongo-csharp-driver/compare/v2.8.0...v2.19.0)

---
updated-dependencies:
- dependency-name: MongoDB.Driver
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

RenanZanelato commented:
Guys, Anyone need some help to resolve this PullRequest? This will resolve a lot my problems !

Commit:
build: updated deps

etiennelepagel committed:
Update azure-pipelines.yaml for Azure Pipelines

Commit:
build: updated deps

ysma500 committed:
fix: Put test project to none in pipelines (#9)

Co-authored-by: Ysmael Fortier <ysma500@ISC-LAP-A6GqDN4.internal.sherwebcorp.com>

delepster committed:
[AUTOMATED] Pull request template update

Commit:
build: updated deps

delepster committed:
fix: remove startup filter and hosted service (#10)

BREAKING CHANGE: configuration methods are removed

Delp1904 committed:
build: Update CODEOWNERS dothraki

Commit:
build: updated deps

Commit:
build: updated deps

Commit:
build: updated deps
